gotocompany/shield/v1beta1/shield.proto

```diff
@@ -227,7 +227,7 @@ service ShieldService {
   rpc UpdateOrganization(UpdateOrganizationRequest) returns (UpdateOrganizationResponse) {
     option (google.api.http) = {
       put: "/v1beta1/organizations/{id}",
-      body: "body"
+      body: "*"
```

Comment:
isn't this replacement making this non backward compatible?

Reply:
seems like I accidentally done this , will recheck and fix